components/network/ntp/test/results-all.master

```diff
@@ -1,3380 +1,107 @@
-3 PASS 
-0 XFAIL 
-0 Failures 
-0 XPASS 
-0 Ignored 
-
-OK
-PASS test-log (exit status: 0)
-
------------------------
-0 Tests: 
-0 PASS 
-0 XFAIL 
-0 Failures 
-0 XPASS 
-0 Ignored 
-
-OK
-PASS test-networking (exit status: 0)
-packetHandling.c:17:test_GenerateUnauthenticatedPacket:PASS
-packetHandling.c:18:test_GenerateAuthenticatedPacket:PASS
-packetHandling.c:19:test_OffsetCalculationPositiveOffset:PASS
-packetHandling.c:20:test_OffsetCalculationNegativeOffset:PASS
-packetHandling.c:21:test_HandleUnusableServer:PASS
-packetHandling.c:22:test_HandleUnusablePacket:PASS
-packetHandling.c:23:test_HandleServerAuthenticationFailure:PASS
-KOD code DENY from 192.0.2.1 192.0.2.1
-packetHandling.c:24:test_HandleKodDemobilize:PASS
-packetHandling.c:25:test_HandleKodRate:PASS
-packetHandling.c:26:test_HandleCorrectPacket:PASS
-
------------------------
-10 Tests: 
-10 PASS 
-0 XFAIL 
-0 Failures 
-0 XPASS 
-0 Ignored 
-
-OK
-PASS test-packetHandling (exit status: 0)
-UnitTest: Incredible packet length: 47.  Discarding.
-UnitTest: Incredible packet length: 47.  Discarding.
-packetProcessing.c:20:test_TooShortLength:PASS
-UnitTest: Incredible packet length: 54.  Discarding.
-UnitTest: Incredible packet length: 51.  Discarding.
-packetProcessing.c:21:test_LengthNotMultipleOfFour:PASS
-UnitTest: Missing extension field.  Discarding.
-packetProcessing.c:22:test_TooShortExtensionFieldLength:PASS
-packetProcessing.c:23:test_UnauthenticatedPacketReject:PASS
-Crypto NAK = 0x00000000 from 0.0.0.0
-packetProcessing.c:24:test_CryptoNAKPacketReject:PASS
-packetProcessing.c:25:test_AuthenticatedPacketInvalid:PASS
-unrecognized key ID = 0x00000032
-packetProcessing.c:26:test_AuthenticatedPacketUnknownKey:PASS
-UnitTest: Packet shows wrong version (0)
-packetProcessing.c:27:test_ServerVersionTooOld:PASS
-UnitTest: Packet shows wrong version (5)
-packetProcessing.c:28:test_ServerVersionTooNew:PASS
-UnitTest: mode 3 stratum 0
-packetProcessing.c:29:test_NonWantedMode:PASS
-packetProcessing.c:30:test_KoDRate:PASS
-packetProcessing.c:31:test_KoDDeny:PASS
-UnitTest: 0.0.0.0 not in sync, skipping this server
-packetProcessing.c:32:test_RejectUnsyncedServer:PASS
-0.0.0.0 response org expected to match sent xmt
-packetProcessing.c:33:test_RejectWrongResponseServerMode:PASS
-packetProcessing.c:34:test_AcceptNoSentPacketBroadcastMode:PASS
-packetProcessing.c:35:test_CorrectUnauthenticatedPacket:PASS
-packetProcessing.c:36:test_CorrectAuthenticatedPacketMD5:XPASS - FAIL WAS EXPECTED, BUT DIDN'T HAPPEN?!
-packetProcessing.c:37:test_CorrectAuthenticatedPacketSHAKE128:PASS
-packetProcessing.c:38:test_CorrectAuthenticatedPacketSHA1:PASS
-packetProcessing.c:39:test_CorrectAuthenticatedPacketCMAC:PASS
-
------------------------
-20 Tests: 
-19 PASS 
-0 XFAIL 
-0 Failures 
-1 XPASS 
-0 Ignored 
-
-FAIL
-PASS test-packetProcessing (exit status: 0)
-=========================================
-   sntp : tests/test-suite.log
-=========================================
-
+FAIL: test-crypto
```

Comment:
With the new `results-all.master` I miss the list of tests.  It is always good to have such a list so we are able to detect when the number of failing tests didn't change, but instead of `test1` failure we now see the `test2` failing.

Reply:
I don't get you here

Reply:
> I don't get you here

Since you reverted this change then it makes little sense to discuss it.